merge.go

```diff
@@ -177,8 +186,18 @@ func mergeConfigMergeArr(opts *options, to, from *Config) Error {
 			field:  fmt.Sprintf("%v", i),
 		}
 
+		// possible for individual index to be replaced
+		idxOpts, err := fieldOptsOverride(opts, "", i)
+		if err != nil {
+			return err
+		}
+		if opts.configValueHandling == cfgReplaceValue {
```

Comment:
should this be idxOpts? Otherwise we can move this if-block up a little.

Reply:
It should have been `idxOpts`, but it not being `idxOpts` meant that the unit test that I thought was testing this path was not. I looked closer at the code and `mergeValues` which does get `idxOpts` passed in, does the correct thing on when set to replace. So this branch was not even needed, I have removed it.